book-example/src/format/config.md

```diff
@@ -71,14 +72,39 @@ The following preprocessors are available and included by default:
 - `index`: Convert all chapter files named `README.md` into `index.md`. That is
   to say, all `README.md` would be rendered to an index file `index.html` in the
   rendered book.
-
+- `emoji`: Convert `:emoji:` text into Emojis. eg: `:smile:` to :smile:
 
 **book.toml**
 ```toml
 [build]
 build-dir = "build"
 create-missing = false
-preprocess = ["links", "index"]
+
+[preprocess]
```

Comment:
This will need to be changed to something like:

```toml
[preprocess.links]

[preprocess.index]

[preprocess.emoji]
```

Although those three preprocessors will be enabled by default, so it's kinda redundant to even mention them...